.github/settings.yml

```diff
@@ -38,9 +58,11 @@ labels:
     color: 2196f3
   - name: test
     color: 8bc34a
+  - name: transversal development
```

Comment:
@Naylin15 These labels was used only in the GLPI plugin, not for all.

Reply:
Ok, I'll remove it when I change the PR and issue template

Just to check the label is only ***transversal development*** or all of the added labels (I ask this since it is hightlined only transversal and you said "these labels")

Reply:
All new labels from the GLPI plugin are only for that workflow, it's a requirement specific from @btry 
If you want, create a hackMD file to review all the labels

Reply:
Transversal development is rarely used. We may consider KISS and drop it, if this can help you.

We may cross issues between 2 repos thanks to hyperlinks